### DISCUSSION TITLE
Filter Component for Table?

### DISCUSSION THREAD

**Discussion by paulwicker** [buefy/buefy#3773] [Q&A]
Has anyone tried designing an Advanced Filter / Quick Filter component for above the table? IMO The #searchable attribute embedding in the column header is a bit clunky when we want to default a table to a pre-filtered view.

**Comment by adamleemiller** [comment]
This would be done pretty easily just using Javascript and filtering the data using a computed method with an input field. If you want to use more than one input field, you will need to ensure that the computed method takes into account all options. We use custom input search fields for filtering all sorts of data, some in tables and some in custom UIs.
